src/portal/item.rs

```diff
@@ -93,6 +93,8 @@ impl Item {
     }
 
     pub(crate) fn encrypt(&self, key: &Key) -> Result<EncryptedItem, Error> {
+        key.strength()?;
```

Comment:
```suggestion
        key.check_strength()?;
```
or something similar maybe?

Reply:
Yeah, that explains more about what it's for.